hack/lib/lint.sh

```diff
@@ -69,7 +70,7 @@ kubeedge::lint::check() {
     echo "check any issue by golangci-lint ..."
     GOOS="linux" golangci-lint run -v
 
-    # only check format issue under staging dir
-    echo "check any issue under staging dir by gofmt ..."
-    gofmt -l -w staging
```

Comment:
The golangci-lint will auto skip the staging dir?

Reply:
Can open an issue in upstream to check the issue why golangci-lint skip the files under staging.

Reply:
https://github.com/golangci/golangci-lint/issues/2409

Reply:
https://github.com/golangci/golangci-lint/issues/2409#issuecomment-991550432

Reply:
according to the golangci-lint upstream community, golangci-lint works as Go: it's module aware.
So it will not check beehive module and viaduct module, we need to check those manually :)